test/include/autoinstall/AutoinstallAskInclude_test.rb

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env rspec
+
+require_relative "../../test_helper"
+
+require "yast"
+
+describe "Yast::AutoinstallAskInclude" do
+  module DummyYast
+    class AutoinstallAskClient < Yast::Client
+      def main
+        Yast.import "Profile"
```

Comment:
I've removed that import but I've added other three (for `Profile`, `UI` and `Stage`) as some of their methods will be mocked.